### PR TITLE
feat(cmd/network/governance): Improve governance proposal display

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: cmd/network/governance/show.go [hunk 1]
```diff
@@ -308,23 +308,34 @@ var (
 				cobra.CheckErr(fmt.Errorf("unexpected proposal state: %v", proposal.State))
 			}
 
+			fmt.Println()
+
+			// Display the proposal content.
+
+			fmt.Println("=== PROPOSAL CONTENT ===")
+			proposal.Content.PrettyPrint(ctx, "", os.Stdout)
```
Comment (matevz) [inline cmd/network/governance/show.go:316]: I'm getting a bunch of parameters listed, but only one was actually changed. Perhaps you could fix the PrettyPrint as well to show only the relevant changes and hide the untouched parameters?

```
Change Parameters:
  Module: staking
  Changes: 
    - Parameter: min_delegation
      Value: <nil>
    - Parameter: reward_schedule
      Value: [map[scale:[1 27] until:90000]]
    - Parameter: min_transact_balance
      Value: <nil>
    - Parameter: fee_split_weight_vote
      Value: <nil>
    - Parameter: fee_split_weight_propose
      Value: <nil>
    - Parameter: reward_factor_epoch_signed
      Value: <nil>
    - Parameter: reward_factor_block_proposed
      Value: <nil>
    - Parameter: min_transfer
      Value: <nil>
    - Parameter: min_commission_rate
      Value: <nil>
    - Parameter: fee_split_weight_next_propose
      Value: <nil>
```

**Review by matevz — APPROVED** [review]
Don't forget to update 
- examples/network-governance/show.out.static
- examples/network-governance/show-votes.out.static.

File: cmd/network/governance/show.go (hunk 1 above)
Comment (kostko) [reply cmd/network/governance/show.go:316]: Unfortunately that pretty print implementation is in Core.

Comment (ptrus) [reply cmd/network/governance/show.go:316]: Pretty print was already fixed in oasis-core, will be fixed once there's an oasis-core release with the fix included (and dep in cli is bumped).

**Pull request merged**